lib/Bio/EnsEMBL/DataCheck/Checks/Denormalized.pm

```diff
@@ -30,7 +30,7 @@ extends 'Bio::EnsEMBL::DataCheck::DbCheck';
 use constant {
   NAME        => 'Denormalized',
   DESCRIPTION => 'Denormalized columns are synchronised',
-  GROUPS      => ['variation', 'schema'],
+  GROUPS      => ['long_running_dc', 'schema'],
```

Comment:
```suggestion
  GROUPS      => ['long_running', 'schema'],
```
The '_dc' isn't necessary (and I tend to avoid the 'dc' abbreviation in any case...)